### DISCUSSION TITLE
Continue to the default Action

### DISCUSSION THREAD

**Discussion by dumanshr** [labwc/labwc#1243] [Ideas]
Currently when I am opening Wofi drun, and click another app. Wofi won't exit. I want to kill wofi when clicking to any frame out of wofi. 
I can implement this. But doing so, mouse event is not passed other app. It only closes wofi and don't do anything in another app. 
I wish to continue default behavior of mouse click or anything after running the first action.

**Comment by johanmalm** [comment]
I think I'm right in saying that with extant protocols, the only way to achieve this is to make the layer-shell client cover the whole output (albeit some transparent) or to implement xdg-popups. I suspect wofi does neither.
